res/values/distribution-strings.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+
+    These strings are used as the SMSSecure description on various distribution channels.
+
+    The sparse numbering of the string variables allows inserting additional lines
+    later without the need to renumber existing strings.
+
+    To translate these strings see the "Contributing Translations" portion of the README
+
+-->
+
+<resources>
+
+    <string name="Distribution_title">SMSSecure</string>
+    <string name="Distribution_description">SMSSecure is an SMS/MMS application that allows you to protect your privacy while communicating with friends.</string>
```

Comment:
F-Droid don't recommand a description with [more than 50 characters](https://f-droid.org/manual/html_node/Summary.html).
Why not "SMS/MMS encryption made easy"?